src/authorizations/components/TokenRow.tsx

```diff
@@ -45,6 +45,7 @@ class TokenRow extends PureComponent<Props> {
     const {auth} = this.props
     const labelText = this.isTokenEnabled ? 'Active' : 'Inactive'
     const date = new Date(auth.createdAt)
+
```

Comment:
👏 easily the best change i've ever seen

😛😛😛

Reply:
lol! I poured blood, sweat, and tears working on line 48.